conf/server/plugin/join_token.conf

```diff
@@ -5,10 +5,5 @@ pluginChecksum = ""
 enabled = true
 pluginType = "NodeAttestor" //needs to match the handshake
 pluginData {
-	join_tokens = {
-		"NOT-A-SECRET"= 60,
-		"THIS-IS-NOT-A-SECRET"= 180,
-		"I-AM-NOT-A-SECRET"= 600
-	}
```

Comment:
Have you tested this? It _should_ be ok since we're not calling the plugin anymore, but want to make sure it still runs without it

Reply:
Good enough considering we don't have functional testing yet :)